modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -699,23 +699,38 @@ class ApiClient(object):
 
         return ss
 
-    def get_authorization_header(self, hdrs, signed_msg):
+    def get_authorization_header(self, signed_headers, signed_msg, created_ts):
         """
         Calculates and returns the value of the 'Authorization' header when signing HTTP requests.
         
-        :param hdrs : The list of signed HTTP Headers
-        :param signed_msg: Signed Digest
+        :param signed_headers : The list of signed HTTP Headers.
+        :param signed_msg: Signed Digest.
+        :param created_ts: The time when the signature was created, as a UNIX timestamp.
         :return: instance of digest object 
         """
 
+        i = 0
+        headers_value = ""
+        is_created_set = False
+        for key, _ in signed_headers.items():
```

Comment:
How about `for i, k in enumerate(signed_headers):`?
Then you don't need to create i, and we never use _ (value so no need to include it in iteration)

Reply:
good suggestion.